ee/codegen/src/__test__/nodes/__snapshots__/subworkflow-deployment-node.test.ts.snap

```diff
@@ -31,8 +42,12 @@ exports[`SubworkflowDeploymentNode > basic > getNodeFile 1`] = `
 
 
 class SubworkflowNode(SubworkflowDeploymentNode):
-    deployment = "58171df8-cdf9-4d10-a9ed-22eaf545b22a"
+    deployment = "test-deployment"
     release_tag = "LATEST"
     subworkflow_inputs = {"input": None}
+
+    class Outputs(SubworkflowDeploymentNode.Outputs):
+        output_1: str
+        output_2: float
```

Comment:
Shows that we now generate outputs and even handle the conversion of kebab-case names into valid  snake case python attribute names.